arcade/arcade/cli/launcher.py

```diff
@@ -364,7 +364,7 @@ def _wait_for_healthy_actor(
 ) -> None:
     """Wait until an HTTP request to `host:port/actor/health` returns 200"""
 
-    while not actor_process.poll():  # Stop waiting if the actor process has exited
+    while actor_process.poll() is None:  # Continue waiting UNLESS the actor process has exited
```

Comment:
This technically fixes it. `poll()` returns the exit code, which was `0` and misinterpreted here as falsey.

`poll() is None` is a better condition that handles any reason the process will exit (zero or non-zero exit codes).